scripts/clusterCreation/deployASB.sh

```diff
@@ -0,0 +1,609 @@
+  # Check the pods
+  kubectl get pods -A
+
+  echo "Completed Getting Cluster Context."
+
+  export ASB_SCRIPT_STEP=setAksVariables
+  # Save environment variables
+  ./saveenv.sh -y
+
+  # Invoke Next Step In Setup
+  $ASB_SCRIPT_STEP
+}
+
+function setAksVariables()
+{
+  echo "Setting AKS Variables..."
+
+  # Set public ip address resource name
+  export ASB_PIP_NAME='pip-'$ASB_DEPLOYMENT_NAME'-'$ASB_ORG_APP_ID_NAME'-00'
+
+  # Get the public IP of our App gateway
+  export ASB_AKS_PIP=$(az network public-ip show -g $ASB_RG_SPOKE --name $ASB_PIP_NAME --query ipAddress -o tsv)
+
+  # Get the AKS Ingress Controller Managed Identity details.
+  export ASB_ISTIO_RESOURCE_ID=$(az deployment group show -g $ASB_RG_CORE -n cluster-${ASB_DEPLOYMENT_NAME}-${ASB_CLUSTER_LOCATION} --query properties.outputs.aksIngressControllerPodManagedIdentityResourceId.value -o tsv)
+  export ASB_ISTIO_CLIENT_ID=$(az deployment group show -g $ASB_RG_CORE -n cluster-${ASB_DEPLOYMENT_NAME}-${ASB_CLUSTER_LOCATION} --query properties.outputs.aksIngressControllerPodManagedIdentityClientId.value -o tsv)
+  export ASB_POD_MI_ID=$(az identity show -n podmi-ingress-controller -g $ASB_RG_CORE --query principalId -o tsv)
+
+  # Get the name of Azure Container Registry
+  export ASB_ACR_NAME=$(az deployment group show -g $ASB_RG_CORE -n cluster-${ASB_DEPLOYMENT_NAME}-${ASB_CLUSTER_LOCATION}  --query properties.outputs.containerRegistryName.value -o tsv)
+
+  # Get Log Analytics Name
+  export ASB_LA_NAME=$(az deployment group show -g $ASB_RG_CORE -n cluster-${ASB_DEPLOYMENT_NAME}-${ASB_HUB_LOCATION} --query properties.outputs.logAnalyticsName.value -o tsv)
+
+  # Get Log Analytics Workspace ID
+  export ASB_LA_WORKSPACE_ID=$(az monitor log-analytics workspace show -g $ASB_RG_CORE -n $ASB_LA_NAME --query customerId -o tsv)
+
+  # Get the name of KeyVault
+  export ASB_KV_NAME=$(az deployment group show -g $ASB_RG_CORE -n cluster-${ASB_DEPLOYMENT_NAME}-${ASB_CLUSTER_LOCATION} --query properties.outputs.keyVaultName.value -o tsv)
+
+  # Config certificate names
+  export ASB_INGRESS_CERT_NAME=appgw-ingress-internal-aks-ingress-tls
+  export ASB_INGRESS_KEY_NAME=appgw-ingress-internal-aks-ingress-key
+
+  echo "Completed Setting AKS Variables."
+
+  export ASB_SCRIPT_STEP=createDnsRecord
+  # Save environment variables
+  ./saveenv.sh -y
+
+  # Invoke Next Step In Setup
+  $ASB_SCRIPT_STEP
+}
+
+function createDnsRecord()
+{
+  echo "Creating Public DNS A Record..."
+
+  # Create public DNS record for ngsa-memory
+  az network dns record-set a add-record -g $ASB_DNS_ZONE_RG -z $ASB_DNS_ZONE -n "ngsa-memory-${ASB_SPOKE_LOCATION}-${ASB_ENV}" -a $ASB_AKS_PIP --query fqdn
+
+  echo "Completed Creating Public DNS A Record."
+
+  export ASB_SCRIPT_STEP=createDeploymentFiles
+  # Save environment variables
+  ./saveenv.sh -y
+
+  # Invoke Next Step In Setup
+  $ASB_SCRIPT_STEP
+}
+
+function createDeploymentFiles()
+{
+  echo "Creating Deployment Files..."
+
+  mkdir -p $ASB_GIT_PATH/istio
```

Comment:
Is this istio templating sections still needed with the recent flux v2 kustomization changes?

Reply:
removed